### ISSUE TITLE
multiple 0x34 pod failures a week after upgrading to Loop v2.3.0 dev

### ISSUE THREAD

**Issue by Lytrix** [LoopKit/Loop#1936]
**Describe the bug**
After upgrading to Loop v2.3.0 dev I have been getting more than normal amount of Pod failures in day 3. 
I first thought it was due to using my older Pods or requesting higher dosages end of day 3, but today I also got a 0x34 of a new lot halfway into day 3 using normal dosages and only a 1,4 bolus.

The failures recorded so far:
2023-01-15 at 2 days, 20 hours (screenshot)
2023-01-18 at 2 days, 23 hours (issue report)
2023-02-22 at 2 days, 23 hours (screenshot)
2023-02-28 at 2 days, 13 hours (issue report)

**Attach an Issue Report**
I've attached the issue reports of 01-18 and 02-28 occurences:
[Loop Report 2023-01-18 19:03:20+01:00.md](https://github.com/LoopKit/Loop/files/10849145/Loop.Report.2023-01-18.19.03.20%2B01.00.md)
[Loop Report 2023-02-28 10:50:33+01:00.md](https://github.com/LoopKit/Loop/files/10849147/Loop.Report.2023-02-28.10.50.33%2B01.00.md)

**To Reproduce**
1. Using a Loop 3 version
2. Higher or even doubling your insulin needs (Being ill for over a month (long covid symptons and on prednisone for a week)
3. Using older pods

**Expected behavior**
I expect to have a 0x34 failure only once each half year, not 1/2 times a week.

**Screenshots**
![2023-02-22 0x34 fault day 3](https://user-images.githubusercontent.com/4370175/221827382-ba11fa2c-04c0-4f46-818d-7f299c82e605.PNG)
![2023-01-15 0x34 fault day 3](https://user-images.githubusercontent.com/4370175/221836389-c7095b8f-5780-4a7f-b851-21a40842040f.PNG)

**Phone**
 - Hardware: Iphone 13 mini
 - OS Version: iOS 16.3.1

**Loop Version**
 - Version Number: v2.3.0
 - Repo: LoopKit/Loop 885cbd3

**CGM**
 - Device: Dexcom G6
 - Manager app: Dexcom App

**Pump**
 - Manufacturer: Omnipod
 - Model: Eros
 - Firmware version: 2.10.0
 
**Additional context**
- Using a Rileylink of 6 months old to connect to Eros 
I think this might be part of what @itsmojo has also seen of having a higher radio on time in comparison with earlier builds.

**Comment by marionbarker** [comment]
There were actually 3 faults in the 2 screenshots and 2 loop reports. I file them by last message date in the logs.

**Comment by ps2** [comment]
Closing this as a duplicate of https://github.com/LoopKit/Loop/issues/1924
